### DISCUSSION TITLE
Testing Kubernetes Instructions

### DISCUSSION THREAD

**Discussion by dnmorris7** [bionic-gpt/bionic-gpt#325] [General]
Spun up a fresh Ubuntu VM and I'm currently testing the [installation instructions ](https://bionic-gpt.com/docs/enterprise-edition/install-linux/)for Kubernetes. I'll keep a log of issues as I come upon them here.

For now, the first issue I encountered. Simple work around of course but disciplined technical documentation leads to better instructional documentation. Leads to happier and less confused users in the long run. 

1. Docker installation required
![image](https://github.com/bionic-gpt/bionic-gpt/assets/7094601/867413a9-38c4-41f8-9249-1027d58ca99f)

2. kubectl wait --namespace ingress-nginx returns an error waiting on timeout.
![image](https://github.com/bionic-gpt/bionic-gpt/assets/7094601/e446e323-1ad6-4f6a-9b3b-b905386e82b1)

Presently looking for a workaround should this step fail. 

3. 503 Service Unavailiable
![image](https://github.com/bionic-gpt/bionic-gpt/assets/7094601/faa1e91d-4a82-4cad-a594-0d67061df905)

After expanding my hard disk partition, I attempted to delete the old cluster and start again. This eliminated the prior issue, however, I'm presently seeing a 503 Service Unavailable Error.

**Comment by ianpurton** [comment]
For changes like this please if you can raise a PR. Thanks.

**Reply by dnmorris7** [reply]
A pull request?

**Reply by ianpurton** [reply]
I fixed it today

**Reply by tbendien** [reply]
@ianpurton what do you mean "a PR"? is that opening an Issue of Git?

**Reply by dnmorris7** [reply]
Cool. Presently my PM is stuck on kubectl wait, but that's clearly on account of my VM no longer having enough disk space. So when I do have it working I'll suggest we mention of how much disk space is necessary to the prerequisite as well.

**Reply by dnmorris7** [reply]
I figure my disk only being 12 GB large by whatever defaults Hyper-V goes by was a major part of the problem. I rectified this and tried rebuilding the cluster. However I now have a 503 error. I presume k9s contains diagnostic features? I'll learn and see what k9s can tell us of this response code next if that's the case.

**Reply by ianpurton** [reply]
Try this https://www.youtube.com/watch?v=ZwOOJqcKQ8g

You'll be able to see the logs etc

**Reply by dnmorris7** [reply]
It might be worth to append to the documentation (under installing on linux) that if there's a permission error when performing `kind get clusters`, that the answer is:

`sudo chown $USER:docker /var/run/docker.sock`
